lib/board.rb

```diff
@@ -0,0 +1,31 @@
+class Board :cells
+
+  def initialize
+    @cells = cells
+  end
+
+  def cells
+    cells = {}
+    x_axis = ("A".."D").to_a
+    y_axis = ("1".."4").to_a
+
+    keys = []
+    x_axis.each do |x|
+      y_axis.each do |y|
```

Comment:
I know Megan hates x and y as variables, but in this case they are block variables and they actually make sense so I'm not sure.